flask_static_digest/__init__.py

```diff
@@ -84,3 +86,28 @@ def static_url_for(self, endpoint, **values):
                                           merged_values.get("filename"))
         else:
             return flask_url_for(endpoint, **merged_values)
+
+
+def parse_digest_host_url(host_url_prefix):
+    """
+    Detect if host_url_prefix contains a path element and returns a tuple with
+    the elements (host_part, path_part), for example for
```

Comment:
Can you re-word this line to be:

`the elements (host_part, path_part), for example:`